MachineLearning/HHMachineLearning/parameters_boosted.py

```diff
@@ -0,0 +1,398 @@
+
+# /!\ Two types of files need to be filled with users parameters 
+#           - parameters.py (mort important one)
+#           - sampleList.py (on what samples to run)
+#           (optionnaly NeuralNet.py for early_stopping etc)
+import multiprocessing
+import os
+from tensorflow.keras.losses import binary_crossentropy, mean_squared_error, logcosh, categorical_crossentropy
+from tensorflow.keras.optimizers import RMSprop, Adam, Nadam, SGD            
+from tensorflow.keras.activations import relu, elu, selu, softmax, tanh, sigmoid
+from tensorflow.keras.regularizers import l1,l2 
+
+from grouped_entropy import GroupedXEnt
+
+##################################  Path variables ####################################
+
+main_path = os.path.abspath(os.path.dirname(__file__))
+path_out = '/nfs/scratch/fynu/gsaha/HHMachineLearning_Boosted_output/'
+path_model = os.path.join(main_path,'model')
+
+scaler_path = '/home/ucl/cp3/gsaha/bamboodev/HHbbWWAnalysis/MachineLearning/HHMachineLearning/scaler_boosted2016_2017_2018.pkl'
+##############################  Datasets proportion   #################################
+crossvalidation = True
+
+# Classic training #
+# -> For crossvalidation == False
+training_ratio = 0.7    # Training set sent to keras
+evaluation_ratio = 0.1  # Evaluation set sent to autom8
+output_ratio = 0.2      # Output set for plotting later
+assert training_ratio + evaluation_ratio + output_ratio == 1 
+# Will only be taken into account for the masks generation, ignored after
+
+# Cross-validation #
+# -> For crossvalidation == True
+N_models = 5  # Number of models to train
+N_train  = 3  # Number of slices on which to train
+N_eval   = 1  # Number of slices on which to evaluate the model
+N_apply  = 1  # Number of slices on which the model will be applied for uses in analysis
+N_slices = N_train+N_eval+N_apply
+splitbranch = 'event' # Will split the training based on "event % N_slices" 
+# /!\ Don't forget to add "splitbranch" in other_variables
+
+if N_slices % N_models != 0: # will not work otherwise
+    raise RuntimeError("N_slices [{}] % N_models [{}] should be == 0".format(N_slices,N_models))
+if N_apply != N_slices/N_models: # Otherwise the same slice can be applied on several networks
+    raise RuntimeError("You have asked {} models that should be applied {} times, the number of slices should be {} but is {}+{}+{}".format(N_models,N_apply,N_models*N_apply,N_train,N_eval,N_apply))
+
+############################### Slurm parameters ######################################
+partition = 'gpu'  # Def, cp3 or cp3-gpu
+QOS = 'normal' # cp3 or normal
+time = '0-06:00:00' # days-hh:mm:ss
+mem = '50000' # ram in MB
+tasks = 1 # Number of threads(as a string) (not parallel training for classic mode)
+cpus = 1
+gpus = 1
+workers = 7
+
+##################################  Naming ######################################
+# Physics Config #
+<<<<<<< HEAD:MachineLearning/HHMachineLearning/parameters_boosted.py
```

Comment:
Did you solve this conflict ?

Reply:
Yes, I did. 
this `parametrers_boosted.py` is a new file with the features for boosted category

Reply:
I can still see the sign of the conflict in the files, either correct, commit and push to add on top of the changes, or I can do it in the next PR very soon